hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1826,6 +1826,48 @@
       interfaces by setting it to 0.0.0.0.</description>
   </property>
 
+  <property>
+    <name>ozone.s3g.content.http.enabled</name>
```

Comment:
These use a common prefix for each component (OM, Recon, etc., see below).  `ozone.s3g.content` is the prefix for this new server, `http` is part of various key suffixes.

We can try to find a better prefix instead of `ozone.s3g.content` (maybe `ozone.s3g.webadmin`).  I'm open to suggestions.

----

```
ozone.om.http-address
ozone.om.http.auth.kerberos.keytab
ozone.om.http.auth.kerberos.principal
ozone.om.http.auth.type
ozone.om.http-bind-host
ozone.om.http.enabled
ozone.om.https-address
ozone.om.https-bind-host

ozone.recon.http-address
ozone.recon.http.auth.kerberos.keytab
ozone.recon.http.auth.kerberos.principal
ozone.recon.http.auth.type
ozone.recon.http-bind-host
ozone.recon.http.enabled
ozone.recon.https-address
ozone.recon.https-bind-host
```

Reply:
@errose28 Do you think `ozone.s3g.webadmin` is a better prefix than `ozone.s3g.content`?  Do you have any alternative suggestion?